hearth/lib/hearth/context.rb

```diff
@@ -34,5 +36,18 @@ def initialize(options = {})
 
     # @return [Hash]
     attr_reader :metadata
+
+    # @return [Array] An ordered list of interceptors
+    attr_reader :interceptors
+
+    def interceptor_context(input, output)
```

Comment:
docs

Reply:
Should this method be documented? Is it private?

Reply:
Added @api private - I dont think this should be considered part of our interface (though it needs to be public to be called from the middelware).